lib/rails_admin/abstract_model.rb

```diff
@@ -60,6 +60,22 @@ def model
       @model_name.constantize
     end
 
+    def quoted_table_name
+      if adapter == :active_record
```

Comment:
Instead of putting a conditional here, you can create the default implementation

```
    def quoted_table_name
      table_name
    end
```

and move the ActiveRecord specific part into [`RailsAdmin::Adapters::ActiveRecord`](https://github.com/railsadminteam/rails_admin/blob/master/lib/rails_admin/adapters/active_record.rb).

Reply:
Fixed.